HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -1270,29 +1271,47 @@ def self.process_load_infiltration_ventilation(mj, hpxml_bldg, all_zone_loads, a
     # Calculate ventilation airflow rates
     q_unb_cfm, q_bal_cfm, q_preheat, q_precool, q_recirc, bal_sens_eff, bal_lat_eff = get_ventilation_data(hpxml_bldg)
 
-    # Calculate adjusted infiltration cfm (combined with ventilation)
-    infil_cfm_heat = (icfm_heat**1.5 + q_unb_cfm**1.5)**0.67 - q_unb_cfm
-    infil_cfm_cool = (icfm_cool**1.5 + q_unb_cfm**1.5)**0.67 - q_unb_cfm
+    # Calculate net infiltration cfm (NCFM; infiltration combined with unbalanced ventilation)
+    if q_unb_cfm > 0
+      # Negative pressure, so NCFM = (ICFM^1.5 + CFMimb^1.5)^0.67
+      infil_ncfm_heat = (icfm_heat**1.5 + q_unb_cfm**1.5)**0.67
+      infil_ncfm_cool = (icfm_cool**1.5 + q_unb_cfm**1.5)**0.67
+    else
+      if icfm_heat < q_unb_cfm.abs
+        # Dominating positive pressure, so NCFM = 0
+        infil_ncfm_heat = 0.0
+      else
+        # Mitigating positive pressure, so NCFM = (ICFM^1.5 - ABS(CFMimb)^1.5)^0.67
+        infil_ncfm_heat = (icfm_heat**1.5 - q_unb_cfm.abs**1.5)**0.67
+      end
+      if icfm_cool < q_unb_cfm.abs
+        # Dominating positive pressure, so NCFM = 0
+        infil_ncfm_cool = 0.0
+      else
+        # Mitigating positive pressure, so NCFM = (ICFM^1.5 - ABS(CFMimb)^1.5)^0.67
+        infil_ncfm_cool = (icfm_cool**1.5 - (-q_unb_cfm)**1.5)**0.67
+      end
+    end
 
-    hpxml_bldg.additional_properties.infil_heat_cfm = infil_cfm_heat
-    hpxml_bldg.additional_properties.infil_cool_cfm = infil_cfm_cool
+    hpxml_bldg.additional_properties.infil_heat_cfm = infil_ncfm_heat
+    hpxml_bldg.additional_properties.infil_cool_cfm = infil_ncfm_cool
 
     # Infiltration load
-    bldg_Heat_Infil = 1.1 * mj.acf * infil_cfm_heat * mj.htd
-    bldg_Cool_Infil_Sens = 1.1 * mj.acf * infil_cfm_cool * mj.ctd
-    bldg_Cool_Infil_Lat = 0.68 * mj.acf * infil_cfm_cool * mj.cool_design_grains
+    bldg_Heat_Infil = 1.1 * mj.acf * infil_ncfm_heat * mj.htd
+    bldg_Cool_Infil_Sens = 1.1 * mj.acf * infil_ncfm_cool * mj.ctd
+    bldg_Cool_Infil_Lat = 0.68 * mj.acf * infil_ncfm_cool * mj.cool_design_grains
 
     # Calculate vent cfm
-    vent_cfm_heat = q_unb_cfm + q_bal_cfm
+    vent_cfm_heat = q_bal_cfm
     vent_cfm_cool = vent_cfm_heat
 
     hpxml_bldg.additional_properties.vent_heat_cfm = vent_cfm_heat
     hpxml_bldg.additional_properties.vent_cool_cfm = vent_cfm_cool
 
     # Calculate vent cfm incorporating sens/lat effectiveness, preheat/precool, and recirc
-    vent_cfm_heat = q_unb_cfm + q_bal_cfm * (1.0 - bal_sens_eff) - q_preheat - q_recirc
-    vent_cfm_cool_sens = q_unb_cfm + q_bal_cfm * (1.0 - bal_sens_eff) - q_precool - q_recirc
-    vent_cfm_cool_lat = q_unb_cfm + q_bal_cfm * (1.0 - bal_lat_eff) - q_recirc
+    vent_cfm_heat = q_bal_cfm * (1.0 - bal_sens_eff) - q_preheat - q_recirc
```

Comment:
I gave it a second thought on how we should calculate the ventilation cfms, there're three cases being considered:
1. supply = exhaust,
    In this case, there's no impact on infiltration, so simply using balanced cfm or total cfm to calculate loads will be good enough.
2. supply < exhaust,
     In this case, home is depressurized, so the infiltration is increased by this amount, the cfm_exhaust - cfm_supply (which is unbalanced cfm) is already accounted in the infiltration calculation, I feel that using balanced cfm here for ventilation will be good.
3. supply > exhaust,
     In this case, the ventilation mitigates infiltration, therefore, infiltration load adjustment doesn't account for the loads caused by bringing more fresh air into the space, I feel that the supply cfm or total cfm should be used.

I was just thinking about the basics here, I didn't find anywhere in the manual J that considers this complexity, let me know if I missed anything.

Reply:
I agree, this is why I say that MJ8 doesn't make any sense. Cases 2 and 3 should be identical, but MJ8 treats them very differently.